opennms-base-assembly/src/main/filtered/bin/opennms

```diff
@@ -634,23 +635,56 @@ doStatus(){
 		return $?
 	fi
 
-	if [ "$VERBOSE" -gt 0 ]; then
-		PARM_VERBOSE=("-v")
-	else
-		PARM_VERBOSE=()
-	fi
-
 	# If there is a PID file for OpenNMS, specify the PID
 	PID="$(getPid)"
 	if [ "$PID" -gt 0 ]; then
-		PARM_PID=("-p" "$PID")
+		# see org.opennms.netmgt.vmmgr.Invoker, we register a custom
+		# signal handler to look for status requests
+		kill -USR1 "$PID"
+
+		touch "/tmp/pre-check.$$"
```

Comment:
is this file doing anything?

Reply:
It's there to be able to do a "newer than" check, to make sure I'm not reading an old status file.  I didn't want to _delete_ the existing status file.  I suppose I could use something along the lines of `find -newermt` but this seemed easiest.

Reply:
Below this, you refer to '${LOG_DIRECTORY}/pre-check.txt' when checking the timestamp if I'm reading it correctly.